.github/workflows/prerelease.yml

```diff
@@ -174,3 +174,39 @@ jobs:
           slack-bot-user-oauth-access-token: ${{ secrets.COREINT_SLACK_TOKEN }}
           slack-channel: ${{ secrets.COREINT_SLACK_CHANNEL }}
           slack-text: "❌ `${{ env.REPO_FULL_NAME }}`: prerelease pipeline failed."
+
+  publish-to-s3:
+    name: Send release assets to S3
+    runs-on: ubuntu-20.04
+    needs: [package-win]
+    steps:
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.OHAI_DOCKER_HUB_ID }}
+          password: ${{ secrets.OHAI_DOCKER_HUB_PASSWORD }}
+      - name: Publish to S3 action
+        uses: newrelic/infrastructure-publish-action@main
```

Comment:
I'd suggest to use a `v1` tag, which is usually overwritten to point to the same commit as `v1.x.y` releases. This way it's easier to keep the action updated and at the same time have a mechanism to protect ourselve from breaking changes.

Reply:
Still pointing at `main`! :)

Reply:
I know :) Let me check if caos is releasing a v1 or not...